README.md

```diff
@@ -52,7 +72,6 @@ And now our element's text is centered.
 
 We can append elements to that element:
 
-
```

Comment:
OK, i'm not horribly opposed to this, but we've not taught looping yet.

Maybe we can say that we're also going to use this as a chance to show looping? Make it less scary?